uber/site_sections/registration.py

```diff
@@ -76,8 +76,7 @@ def index(self, session, message='', page='0', search_text='', uploaded_id='', o
             'order':          Order(order),
             'attendee_count': total_count,
             'checkin_count':  session.query(Attendee).filter(Attendee.checked_in != None).count(),
-            'attendee':       session.attendee(uploaded_id) if uploaded_id else None,
-            'remaining_badges': max(0, c.MAX_BADGE_SALES - c.BADGES_SOLD)
+            'attendee':       session.attendee(uploaded_id) if uploaded_id else None
         }
```

Comment:
interestingly enough, ```remaining_badges``` seems to not currently be used by index.html template.  I seem to remember at once point it might have been.  regardless we don't need this here anymore whee